docs/TUBITAK-doc/ProjeÖzeti.md

```diff
@@ -0,0 +1,57 @@
+
+ - Ödevle eğitimin yarım kalmasına neden olan öğrencinin geri dönüt alamaması sorununu ortadan kaldırır,
+
+ - Öğrencinlerin sadece ödev yaparak değil aynı zamanda ödev kantrol ederek te öğrenmesini sağlar, 
+
+ - Öğrencilerin notlarının adil bir şekilde dağıtılmasını sağlar.<br/><br/><br/><br/>
+
+
+# SUMMARY of PROJECT - [EN]
+
+
+The high number of students in some departments in universities affects teaching negatively when it comes to homeworks. The contribution of homework to teaching is an indisputable fact. Not only doing the homework, but also checking the homework is as instructive as doing the homework. However, the lack of assistants and/or the high number of students cause difficulties in assigning homework for teachers. SOTAS aims to eliminate these difficulties and to establish a system that teaches both by doing homework and by checking homework and to solve the problems bilaterally.
+
+When students studying in crowded classrooms are given more than one homework, it becomes difficult to get feedback on their questions about homework. Teachers have a hard time getting back to students because they spend too much time checking homework. At this point, education is interrupted because the purpose of the homework is for the student to have knowledge about the subject by researching the given task and to improve himself. It is the acquisition of information through the questions asked by the teacher in case the information obtained is not sufficient.
+
+This situation has still not been resolved in educational institutions with crowded classrooms and limits the education of the student. At this point, the importance of our project is understood more. In addition to solving all these problems, SOTAS also increases the quality of education by having students check their homework, allowing them to take a look at the wisdom they have acquired, and obtaining more information from their homework than they can acquire on their own.
+
+One person checking the homework of more than one person causes the fair environment to deteriorate at some point. SOTAS is a system where the student knows that he/she will be scored by others and therefore pushes him/her to give fair grades. Thus, it also ensures that the student is honest and fair.
+
+If we need to collect all these pluses of SOTAS in one place, the project is;
+
+ - Eliminates the problems of excess student and lack of assistants, which are the main reasons why teachers do not give homework,
+
+ - Eliminates the problem of the student not getting feedback, which causes the education to be incomplete with homework,
+
+ - It allows students to learn not only by doing homework, but also by checking homework,
+
+ - It ensures the fair distribution of students' grades.
+
+
+
+
+
+SOTAS, öğrencilerin ödev yapmasını ve diğer arkadaşlarının ödevlerini değerlendirerek puanlayıp neden o puanı verdiğine dair yorum yapmasını sağlayan bir platformdur.
```

Comment:
What this text for?

Reply:
Imposter turkish sentence just looking around and investigating english sentences.

Reply:
when the imposter is sus